### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
Add support for Python 3.5, 3.7, 3.8 and 3.9.

### CODE REVIEW

File: src/zc/table/batching.py [hunk 1]
```diff
@@ -57,20 +57,20 @@ def setPrefix(self, prefix):
         super(Formatter, self).setPrefix(prefix)
         self._batch_start_computed = False
 
-    @apply
```
Comment (mgedmin) [inline src/zc/table/batching.py:60]: How did this package ever claim Python 3 support?  `apply` does not exist in Python 3!

Does zc.table.batching have no tests at all?

**Review by mgedmin — APPROVED** [review]
LGTM

File: src/zc/table/batching.py (hunk 1 above)
Comment (mgedmin) [reply src/zc/table/batching.py:60]: Yeah!

```
Name                                                                Stmts   Miss  Cover
---------------------------------------------------------------------------------------
.tox/coverage/lib/python3.7/site-packages/zc/table/__init__.py          2      0   100%
.tox/coverage/lib/python3.7/site-packages/zc/table/batching.py         76     76     0%
```

Zero.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**